### ISSUE TITLE
Include example screenshots in documentation

### ISSUE THREAD

**Issue by evnb** [TkinterEP/ttkwidgets#65]
As a prospective user of ttkwidgets, it would be very helpful to have example screenshots to see how these widgets look, and if they align with my desired functionality. I imagine I'm not alone in this. 

Not a priority but it would be nice :)

Here is an example of this in another set of widgets: [https://github.com/slightlynybbled/tk_tools](https://github.com/slightlynybbled/tk_tools)

**Comment by WinEunuuchs2Unix** [comment]
@envb I agree wholeheartedly but it seems most python documentation doesn't have screenshots. That said here is a screenshot for ttkwidgets' class CheckboxTreeview():

![CheckboxTreeview selections](https://user-images.githubusercontent.com/33849503/97088510-899b7b00-15ee-11eb-90c3-27b666adfaeb.png)

I took the screenshot because I think the "checked" checkbox image is too subtle. On some of my monitors it is difficult to differentiate from the "unchecked" checkbox image.

My current work around is to set the "checked" background color to light steel blue 1.

I don't know much about **Gimp** but I'm thinking of trying to set colors to checkbox images such that:

- "checked" is a green checkmark ✔
- "unchecked" is a red cross ❌
- "tristate" is a blue square 🟦 

Not to sure if this will be successful.

Too make a long story short screenshots in the first place would help you see what you will be getting. In the second place it makes it easier for people to discuss the widget and discuss changes to it.

**Comment by RedFantom** [comment]
This is certainly something to be considered. However, personally, I would very much prefer if the screenshots could be taken by a script, so there would be no need to manually recreate them every time a widget changes. However, some widgets offer advanced functionality, so that would require adding some sort of action system before taking the screenshots. This makes it quite complicated to do, so that's why there currently are no screenshots included.

In the meantime, I'm not opposed to adding screenshots for each widget, but it isn't a priority right now. The example scripts give the best impression of what the widgets can do, and I recommend giving them a go if you are interested in a particular widget.

**Comment by rdbende** [comment]
I sat down and took a few screenshots because I noticed that the user is greatly influenced by whether they immediately get an image of what the library looks like (my Azure theme has also become so popular in a short time because I attached a cool screenshot).

I'm too lazy to open a pull request, but I would be happy if you could incorporate this into README.md.
 
It became a little bit too Windowsic, sorry Linux and Mac users :(
![askcolor](https://user-images.githubusercontent.com/77941087/109359663-26647280-7886-11eb-9f09-c69f4871b7da.png)
![askcolor_withalpha](https://user-images.githubusercontent.com/77941087/109359664-26fd0900-7886-11eb-9df4-a7300c5873ae.png)
![askfont](https://user-images.githubusercontent.com/77941087/109359665-26fd0900-7886-11eb-9c1f-131d80a70982.png)
![autocombo](https://user-images.githubusercontent.com/77941087/109359666-26fd0900-7886-11eb-9899-0368027df171.png)
![autoentry](https://user-images.githubusercontent.com/77941087/109359667-27959f80-7886-11eb-9117-3024a33e7645.png)
![autoentrylistbox](https://user-images.githubusercontent.com/77941087/109359669-27959f80-7886-11eb-9b3b-c546e66d4015.png)
![autohidescrollbar](https://user-images.githubusercontent.com/77941087/109359670-27959f80-7886-11eb-8a1b-07fd6a2402b3.png)
![calendar](https://user-images.githubusercontent.com/77941087/109359671-282e3600-7886-11eb-936c-35afd91c90e2.png)
![checkbowtreeview](https://user-images.githubusercontent.com/77941087/109359672-282e3600-7886-11eb-9eea-5b1a80149ba9.png)
![debugwindow](https://user-images.githubusercontent.com/77941087/109359674-28c6cc80-7886-11eb-9d72-94d280067baa.png)
![fontchooser](https://user-images.githubusercontent.com/77941087/109359675-28c6cc80-7886-11eb-805c-030b543c5d49.png)
![fontselectframe](https://user-images.githubusercontent.com/77941087/109359676-28c6cc80-7886-11eb-92a8-be6f71ea164f.png)
![itemscanvas](https://user-images.githubusercontent.com/77941087/109359677-295f6300-7886-11eb-9caa-692d32485dbe.png)
![linklabel](https://user-images.githubusercontent.com/77941087/109359680-295f6300-7886-11eb-97f6-db59515fcfb9.png)
![scaleentry](https://user-images.githubusercontent.com/77941087/109359682-295f6300-7886-11eb-81a2-3d4f1a58208b.png)
![scrolledframe](https://user-images.githubusercontent.com/77941087/109359683-29f7f980-7886-11eb-9a5c-8a7d89b97faa.png)
![scrolledlistbox](https://user-images.githubusercontent.com/77941087/109359685-29f7f980-7886-11eb-869e-4bc30aa4504e.png)
![table](https://user-images.githubusercontent.com/77941087/109359686-29f7f980-7886-11eb-83d7-2657d2c0f95c.png)
![tickscale](https://user-images.githubusercontent.com/77941087/109359687-2a909000-7886-11eb-9a8e-25e42db07daa.png)
![timeline](https://user-images.githubusercontent.com/77941087/109359688-2a909000-7886-11eb-92dc-b3661f8a959d.png)
![toggledframe](https://user-images.githubusercontent.com/77941087/109359689-2b292680-7886-11eb-904e-7c7de9253fde.png)
![tooltip](https://user-images.githubusercontent.com/77941087/109359690-2b292680-7886-11eb-8c28-31bebcb8aecc.png)
